common_examples/iperf/main/iperf.c

```diff
@@ -0,0 +1,80 @@
+#include "esp_netif.h"
+#include "esp_eth.h"
+#include "esp_event.h"
+#include "esp_console.h"
+#include "ethernet_init.h"
+#include "iperf_cmd.h"
+#include "sdkconfig.h"
+
+esp_netif_ip_info_t ip_info = {
+    .ip = {.addr = ESP_IP4TOADDR(192, 168, 1, 1)},
+    .netmask = {.addr =  ESP_IP4TOADDR(255, 255, 255, 0)},
+    .gw = {.addr = ESP_IP4TOADDR(192, 168, 1, 1)}
+};
+
+static void start_dhcp_server_at_connection(void *esp_netif, esp_event_base_t base, int32_t event_id, void *data)
+{
+    esp_netif_dhcpc_stop(esp_netif);
+    esp_netif_set_ip_info(esp_netif, &ip_info);
```

Comment:
If we have device with 2 NICs and DHCP server, do they both NICs have the same IP?

Reply:
From the tests I ran - yes. Both devices which I connected to network interfaces were able to refer to the server by the same IP address, and each got assigned own IP address.

Reply:
That probably works but I'm not sure how the lwIP decides which NIC to use when transmitting when both NICs have the same IP address.

Reply:
I think they should be in different subnets.

Reply:
Will experiment with this separately, and make and additional commit on success.

Reply:
I've managed to make this work after adding `ESP_NETIF_FLAG_AUTOUP` as seen here: https://github.com/espressif/esp-idf/issues/4086#issuecomment-555903861

Reply:
The issue I was facing with implementing it was resolved.

Reply:
Oh, right, I planned on updating `iperf` after the code in `tcp_server` is deemed good in order to prevent updating two files each time a change needs to be made. If no more changes are needed I'll update `iperf` accordingly.